pep/app/pods/components/page/drawer/component.ts

```diff
@@ -26,6 +27,10 @@ export default class PageDrawer extends Component<PageDrawerArgs> {
         return this.configuration.defaultSearchParams;
     }
 
+    get canLogOut() {
```

Comment:
since this getter is duplicated in two places and we have our own session service now, I think it would be nice if we just defined this directly in PepSession, so you can just do `this.session.canLogOut`.